walkthrough/src/modules/app/walkthrough/_javascript.jade

```diff
@@ -142,7 +142,7 @@ questions.Questions--2
       answer-toggle(correct="false")
         :markdown
           ```
-          .SearchForm .isSuperUser.Btn {
+          .SearchForm .isSuperUser .Btn {
```

Comment:
yes i understand that!, i made this two last changes only for give a similar shape to the allowed selectors, even if the last one are not allowed, this will help to understand to not directly styling . isSuperUser because it breaks module encapsulation.